external/precompiled/linux/include/zconf.h

```diff
@@ -0,0 +1,534 @@
+/* zconf.h -- configuration of the zlib compression library
```

Comment:
I'm so sorry to keep dragging out your PR, but shouldn't the build find zconf.h for your build like it finds zconf for the 32-bit and 64-bit x86 builds?